monitor/services/src/main/java/org/datacleaner/monitor/configuration/ConfigurationCache.java

```diff
@@ -106,8 +107,12 @@ protected DataCleanerConfiguration decorateConfiguration(DataCleanerConfiguratio
     }
 
     protected DataCleanerConfiguration readConfiguration() {
+        return readConfiguration(null);
+    }
+    
+    DataCleanerConfiguration readConfiguration(Map<String, String> overrideProperties) {
         final ConfigurationReaderInterceptor interceptor = new MonitorConfigurationReaderInterceptor(_repository,
-                _tenantContext, _injectionManagerFactory);
+                _tenantContext, overrideProperties, _injectionManagerFactory);
```

Comment:
Hmmm... I don't like it being part of the cache, but I don't really like the alternative either.

The best would be to get the configuration first (cached), and the apply overrides after, but the interceptor design pretty much precludes that... :(

Oh well, leave it. :)